options/options.html

```diff
@@ -36,7 +43,10 @@
         When a URL of a pinned tab matches one of these case insensitive <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions">regular expressions</a>, its links will not be changed. Use this to ignore sites that already behave like pinned apps. Example: <code>twitter\.com</code>
       </p>
       <div class="info"><span></span>&nbsp;</div>
-      <textarea class="optionValue" rows="10" cols="50"></textarea>
+      <textarea id="option" rows="10" cols="50" class="optionValue"></textarea>
+
+      <p class="title"><b>Behaviour</b></p>
+      <label><input type="checkbox" id="optionStickDomain" class="optionValue" value="checked"> Open internal links in the pinned tab</label>

```

Comment:
I think this should not be checked by default. It definitely depends on the site to make this useful. For example, when pinning [the GitHub review queue](https://github.com/pulls/review-requested), opening internal links within the pinned tab loses the queue. It makes more sense to open internal links in a new tab for that one (IMO).

Reply:
The value is perhaps confusing here: checkboxes are checked through the `checked` property, and typically in a form context the `value` is only submitted when `checked === true`. This is emulated on restore in `options/options:js:9`. And on save at `options/options.js:39`. So the value of the checkbox isn't used unless the user checks the checkbox.

Reply:
Ah the value isn't used anywhere, I'm removing it.